x/ethbridge/types/msgs.go

```diff
@@ -17,6 +17,36 @@ const (
 	lockGasCost = 60000000000 * 393000
 )
 
+var _ sdk.Msg = &MsgPause{}
+
+// Route should return the name of the module
+func (msg MsgPause) Route() string { return RouterKey }
+
+// Type should return the action
+func (msg MsgPause) Type() string { return "pause" }
+
+// ValidateBasic runs stateless checks on the message
+func (msg MsgPause) ValidateBasic() error {
+	if msg.GetSigner() == "" {
+		return sdkerrors.ErrInvalidAddress
+	}
+	return nil
+}
+
+// GetSignBytes encodes the message for signing
+func (msg MsgPause) GetSignBytes() []byte {
+	return sdk.MustSortJSON(ModuleCdc.MustMarshalJSON(&msg))
+}
+
+// GetSigners defines whose signature is required
+func (msg MsgPause) GetSigners() []sdk.AccAddress {
+	addr, err := sdk.AccAddressFromBech32(msg.Signer)
```

Comment:
We should probably use MustAccAddressFromBech32 here as it panics on failure rather than rewriting it ourselves

Reply:
@smartyalgo I Just realized the integration tests using siftool may not be included in the automated tests for the PR. Can you verify the .github file has been updated to use the new siftool tests?

Reply:
Updated to use MustAccAddressFromBech32.
Added to github action